target/linux/octeon/base-files/lib/preinit/79_move_config

```diff
@@ -1,19 +1,23 @@
 # Copyright (C) 2014 OpenWrt.org
 
-move_config() {
-	. /lib/functions.sh
+octeon_move_config() {
 	. /lib/upgrade/common.sh
```

Comment:
Can you please preserve the empty lines after the includes?

Reply:
sure, thanks.
added newlines after the includes.